### PR TITLE
[INFRA-4591] Up deps

### PR DESCRIPTION
**Pull request by lord-nibbler**
use liberation font, which comes with gonum pkg:
image

### CODE REVIEW

**Comment by sazap10** [comment]
i guess this PR supersedes https://github.com/bugsnag/promalert/pull/38

**Comment by lord-nibbler** [comment]
ah yes

File: docker-compose.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-version: '3.7'
+version: '3.9'
```
Comment (sazap10) [inline docker-compose.yml]: we shouldnt need the version anymore

**Review by sazap10 — DISMISSED** [review]
Looks reasonable

**Review by sazap10 — APPROVED** [review]

**Pull request merged**